contributors/devel/sig-testing/verify-tests.md

```diff
@@ -81,3 +82,44 @@ In order of decreasing preference:
 
 Analysis configuration can be found at [kubernetes/kubernetes/hack/testdata/levee/levee-config.yaml](https://github.com/kubernetes/kubernetes/blob/master/hack/testdata/levee/levee-config.yaml).
 Contact SIG-Security with any additional questions.
+
+### `verify-api-groups`
+
+This verification script validates the different api-groups by reading
+the respective `register.go` file. Every register file should contain
+a GroupName from where the different representations can be
+gathered. Also, the other representations will be, `group_dirnames`,
+`external_group_versions`, `expected_install_packages`.
+
```

Comment:
Should implies convention, use 'must' instead if it's a hard requirement.  Remove implementation details and extra blank line
```suggestion
the respective `register.go` file. Every register file must contain
a `GroupName`.
```